requirements-base.txt

```diff
@@ -34,7 +34,7 @@ progressbar2==3.32.0
 # If we bump it to 2.8.4 or greater then Python 3.6 & 3.8 would use the same version
 psycopg2-binary==2.7.7; python_version == '3.6'
 psycopg2-binary==2.8.6; python_version > '3.6'
-PyJWT==1.5.3
+pyjwt[crypto]==2.0.1
```

Comment:
Have you reviewed the changelog here? I don't know what we use this for, but bumping a major version is generally risky.

Reply:
Yes, I didn't want to do this but I had to, for some reason when generating JWTokens using 1.5.3 app store connect rejects the credentials (it took me a while to figure out that an upgrade fixed the problem and there was not something wrong in how I generated the tokens).

Reply:
I mean, have you verified that this doesn't break other places that use this library? I see many broken tests that look related.

Reply:
Yes, I still need to figure out the broken tests, obviously will do before merging.

Reply:
It might be safer to upgrade this separately and make sure our existing code is compatible, then introduce your new feature in this pr. Otherwise this pr is going to end up getting pretty large, I think.